pkg/operator/apis/monitoring/v1/monitoring_types.go

```diff
@@ -44,10 +44,10 @@ type MonitoringCondition struct {
 	Status corev1.ConditionStatus `json:"status"`
 	// The last time this condition was updated.
 	// +optional
-	LastUpdateTime metav1.Time `json:"lastUpdateTime,omitempty"`
+	LastUpdateTime metav1.Time `json:"lastUpdateTime"`
```

Comment:
Hm, I think those are not safe to remove. Modernize is doing this assuming we use std JSON encoder/decoder but I wonder if that's true (we use yaml.v3, same k8s).